src/Microsoft.AspNet.Mvc.Core/Formatters/OutputFormatter.cs

```diff
@@ -237,21 +239,17 @@ private Encoding MatchAcceptCharacterEncoding(IList<StringWithQualityHeaderValue
             if (acceptCharsetHeaders != null && acceptCharsetHeaders.Count > 0)
             {
                 var sortedAcceptCharsetHeaders = acceptCharsetHeaders
-                                                    .Where(acceptCharset =>
-                                                                acceptCharset.Quality != HeaderQuality.NoMatch)
-                                                    .OrderByDescending(
-                                                        m => m, StringWithQualityHeaderValueComparer.QualityComparer);
+                    .Where(acceptCharset => acceptCharset.Quality != HeaderQuality.NoMatch)
+                    .OrderByDescending(m => m, StringWithQualityHeaderValueComparer.QualityComparer);
 
                 foreach (var acceptCharset in sortedAcceptCharsetHeaders)
                 {
                     var charset = acceptCharset.Value;
                     if (!string.IsNullOrWhiteSpace(charset))
                     {
-                        var encoding = SupportedEncodings.FirstOrDefault(
-                                                        supportedEncoding =>
-                                                            charset.Equals(supportedEncoding.WebName,
-                                                                           StringComparison.OrdinalIgnoreCase) ||
-                                                            charset.Equals("*", StringComparison.Ordinal));
+                        var encoding = SupportedEncodings.FirstOrDefault(supportedEncoding =>
+                            charset.Equals(supportedEncoding.WebName, StringComparison.OrdinalIgnoreCase) ||
+                            charset.Equals("*", StringComparison.Ordinal));
```

Comment:
There is in HttpAbstraction's `MediaTypeHeaderValue` and MVC's `MediaTypeHeaderValueComparer+ParsedMediaTypeHeaderValue` but both are `private`.